index.html

```diff
@@ -138,17 +138,17 @@ <h3 id="scenario-level">Scenario level</h3>
         terrain <img class="icon" src="https://gloomhaven-secretariat.de/assets/images/terrain.svg"></strong>
       are
       shown. The scenario
-      level is automatically set based on the added character level.
+      level is automatically set based on the level of the added characters.
     </p>
 
-    <p>To adjust difficulty set <strong>Scenario level adjustment</strong> in level menu.</p>
+    <p>To adjust the difficulty, click or touch the values, then use <strong>Scenario level adjustment</strong>.</p>
 
-    <p>To change the level manually just click on the values and first disable <strong>Automatic level
-        calculation</strong>, afterwards select the corresponding level. The recommended level based on the
-      character's
+    <p>To change the level manually, first disable <strong>Automatic level
+        calculation</strong>, then select the corresponding level. The recommended level based on the
+      characters'
       level is highlighted with a lighter
-      circle if not matching the current level. (To change character levels, see <a href="#updating-character">Updating
-        character</a> or additional <a href="#managing-character">Managing character</a>.)</p>
+      circle if it does not match the current level. (To change character levels, see <a href="#updating-character">Updating
+        Character</a> and <a href="#managing-character">Managing Character</a>.)</p>
 
     <p>With <strong>Level bonuses adjustment</strong> set adjustment of the level used for bonuses calculation and
       <strong>&ge; 5 player rule</strong> (Only visible by &ge; 5 characters) apply Isaac's 5 player rules (Level+2,
```

Comment:
I also had a hard time understanding this sentence about Isaac's 5-player rules, but I left it in. Can you point me to where this is described elsewhere?

Reply:
https://boardgamegeek.com/thread/1702348/article/24683436#24683436

https://boardgamegeek.com/thread/1702348/article/24685523#24685523